cmd/osctl/cmd/config.go

```diff
@@ -145,12 +151,14 @@ var configGenerateCmd = &cobra.Command{
 			if err = writeUserdata(input, udType, "master-"+strconv.Itoa(idx+1)); err != nil {
 				helpers.Fatalf("failed to generate userdata for %s: %v", "master-"+strconv.Itoa(idx+1), err)
 			}
+			fmt.Println("created file", workingDir+"/master-"+strconv.Itoa(idx+1)+".yaml")
```

Comment:
should this go to `stderr`?

Reply:
I didn't _think_ so. I followed what we do with the output for `osctl cluster create`.